2006-06-18  Mike Frysinger  <vapier@gentoo.org>

	* sysdeps/unix/sysv/linux/getcwd.c: Include sys/param.h.

--- glibc-old/sysdeps/unix/sysv/linux/getcwd.c
+++ glibc-new/sysdeps/unix/sysv/linux/getcwd.c
@@ -24,6 +24,7 @@
 #include <limits.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <sys/param.h>
 
 #include <sysdep.h>
 #include <sys/syscall.h>
